client/src/main/java/io/pravega/client/ClientConfig.java

```diff
@@ -73,6 +75,9 @@ public ClientConfig build() {
                 controllerURI = URI.create("tcp://localhost");
             }
             extractCredentials();
+            if (credentials == null) {
+                log.info("The credentials are not specified/could not be extracted");
```

Comment:
This is an info message but the change says 'warning'. These should be made consistent one way or the other.

Reply:
Fixed.